stylesheets/styles.css

```diff
@@ -0,0 +1,471 @@
+#layer_1{
+    fill: #fff;
+    height: 60px;
+    width: 396px;
+}
+a {
+    color: inherit;
+    text-decoration: none;
+}
+ul{
+    padding:0;
+    margin-top:17px;
+    margin-bottom:20px;
+    
+    padding-bottom: 10px;
+    
+    padding-top: 15px;
+}
+.byline{
+    font-size: 14px;
+    line-height: 21px;
+    color: #555;
+    text-transform: uppercase;
+    margin-top: 4px;
+    font: bold 17px/21px "Roboto Condensed",sans-serif;
```

Comment:
The use of this shorthand overwrites the `font-size` and line-height properties on lines 20 and 21